::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2018 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Fri, 17 Aug 2018 17:20:13 +0200
Subject: [PATCH] Switch back to 4.4-based LTS kernel

JIRA: ARMBAND-398

Change-Id: I329346755348525f00602d7f89af295c141a20d5
Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 mcp/config/scenario/defaults.yaml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/mcp/config/scenario/defaults.yaml b/mcp/config/scenario/defaults.yaml
index f5640f71..b228463c 100644
--- a/mcp/config/scenario/defaults.yaml
+++ b/mcp/config/scenario/defaults.yaml
@@ -43,7 +43,6 @@ aarch64:
         - armband 1201 deb [arch=arm64] http://linux.enea.com/mcp-repos/queens/xenial queens-armband main
     pkg:
       install:
-        - linux-generic-hwe-16.04
         - python-futures
         - salt-minion
   control:
